### PR TITLE
Add "restrictions on use" to the fork section

### PR DESCRIPTION
**Pull request by CrystalBallBe**

### CODE REVIEW

File: collections/_version2-english/9-fork/3-restrictions-on-use.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+
+Even if a market in the parent universe has not been migrated to the [winning universe]({{glossary_winning_universe}}), users may trade and [settle]({{glossary_settlement}}) their [shares]({{glossary_share}}) on the market. It does not matter whether the market is already finalized or not, users can do those on any market in the parent universe. Trading in the parent universe, traders pay [creator fee]({{glossary_creator_fee}}) and [reporting fee]({{glossary_reporting_fee}}), and those fees are added to [reporting fee pool]({{glossary_reporting_fee_pool}}). So users may purchase [participation tokens]({{glossary_participation_token}}) to collect those fees and redeem them.
+
+Users also may create a market in the parent universe. However, after a [fork]({{glossary_fork}}) started, a market in the parent universe never been finalized. To finalize a market, someone needs to migrate it to the winning universe (See [migration of the objects]({{url_migration_of_the_objects}}#non-forking-market) for details).
```
Comment (MicahZoltu) [inline collections/_version2-english/9-fork/3-restrictions-on-use.md:141]:
```suggestion
Users may also create a market in the parent universe. However, after a [fork]({{glossary_fork}}) has started, a market in the parent universe can not be finalized. To finalize a market, someone needs to migrate it to the winning universe (See [migration of the objects]({{url_migration_of_the_objects}}#non-forking-market) for details).
```

File: collections/_version2-english/9-fork/3-restrictions-on-use.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+
+In child universes, there are no restrictions. It doesn't matter whether a child universe is winning one or losing one. You can do everything on them as soon as they are created. (Note: child universes are not created as soon as a fork starts. They are created when the first time REP is migrated to them. See [migration of the objects]({{url_migration_of_the_objects}}#forking-market) for details.)
```
Comment (MicahZoltu) [inline collections/_version2-english/9-fork/3-restrictions-on-use.md:143]:
```suggestion
In child universes, there are no restrictions. It doesn't matter whether a child universe is winning one or losing one. You can do everything on it as soon as it is created.  Note: child universes are not created as soon as a fork starts, they are created the first time REP is migrated to them.  See [migration of the objects]({{url_migration_of_the_objects}}#forking-market) for details.)
```

**Review by MicahZoltu — APPROVED** [review]
Can you really create new markets in the parent universe during a fork?  I thought you couldn't?

**Comment by CrystalBallBe** [comment]
> Can you really create new markets in the parent universe during a fork? I thought you couldn't?

@MicahZoltu Yes, I think you can.  But I don't verify that, so I'm going to try to verify that.

FWIW, Initially, I thought you could not create, but when you reviewed the table I made (it's a long time ago), you said that you should still be able to create markets during the fork.
Reference: https://docs.google.com/spreadsheets/d/1CmBMDPbTvgFcBYYpV6TVFQU8vEmxN3o40zpVN6IIQUE/edit?disco=AAAAG7Gf4mg

**Comment by CrystalBallBe** [comment]
I read the contracts, and now I believe you cannot create new markets in the parent universe during a fork. Because of [this requirement](https://github.com/AugurProject/augur/blob/54099c1260e57ba3b6f5d4c506d697d119dd9624/packages/augur-core/src/contracts/factories/MarketFactory.sol#L46).
...Is my guess correct?

**Comment by MicahZoltu** [comment]
Good find!  That definitely looks like you cannot create a market during a fork!

**Comment by CrystalBallBe** [comment]
I’m closing this PR, and after that, I will submit another PR which includes the suggestions @MicahZoltu submitted here. Because I cannot merge this PR due to conflicts, and I don't know how to resolve that.

**Comment by CrystalBallBe** [comment]
I reflected the suggestions in the PR https://github.com/msagansk/Augur.Guide/pull/45

**Pull request closed without merging**